ansible/roles/kernel/tasks/os/debian.yml

```diff
@@ -30,24 +30,12 @@
 - block:
   - name: Reboot
     reboot:
```

Comment:
nit: fully qualified names would be nice here since formatting is already a little odd.